.eslintignore

```diff
@@ -1,4 +1,5 @@
 **/node_modules/*
 **/dist/*
+android/**/build/*
```

Comment:
This fixes `npm run lint` that I reported two weeks ago [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1707390669065259)